library/src/yaml.rs

```diff
@@ -12,6 +12,8 @@ pub struct YamlConfig {
     pub base_url: Option<String>,
     /// Update behavior. Defaults to true if not set.
     pub auto_update: Option<bool>,
+    /// Path to a public key that will be used to verify patch hash signatures.
+    pub patch_public_key_path: Option<String>,
```

Comment:
I'm confused by this path because shorebird.yaml on disk vs. shorebird.yaml in the apk are different formats.

Does this point to the path on disk during development and an apk path during production?